scene/3d/mesh_instance_3d.cpp

```diff
@@ -327,7 +327,10 @@ void MeshInstance3D::create_multiple_convex_collisions(const Ref<MeshConvexDecom
 
 void MeshInstance3D::_notification(int p_what) {
 	switch (p_what) {
-		case NOTIFICATION_ENTER_TREE: {
+		case NOTIFICATION_UNPARENTED: {
+			skeleton_path = NodePath();
```

Comment:
This will reset it when reparenting by manual use, this will be very surprising and confusing, why is it needed?

Reply:
What I am trying to do is to prevent the node from using the old value of `skeleton_path` when the relative path to the skeleton may have changed after reparenting.

Reply:
But you can remove it and then add it back and expect it to work

Reply:
And in that case, if the path is invalid, it should still signal an error